Bottom: c2570d877e4b6274a84b76e14af49deef5eab0d6
Top:    25e55e65529eaa358689f5347776e7ac253ee827
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-23 23:43:19 +0000

ODDS AND SODS


---

diff --git a/src/polygon.h b/src/polygon.h
index aabc096..1343d03 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 10 //40
+#define POLY_CIRC_SEGS 8 //10 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
